app/spiffworkflow/messages/MessageHelpers.js

```
@@ -318,34 +318,34 @@ export function createOrUpdateCorrelationProperties(
+              expr.message_ref,
             );
             retrievalExpression.messageRef = msgElement;
             correlationProperty.correlationPropertyRetrievalExpression.push(
-              retrievalExpression
+              retrievalExpression,
             );
           }
         });
 
         const existingIndex = definitions.rootElements.findIndex(
           (element) =>
```

ESLint

[error] 'element' is already declared in the upper scope on line 287 column 3.

(no-shadow)